charts/custom-distribution-service/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: 0.1-alpha-1
+appVersion: custom-distribution-service-0.0.2
```

Comment:
i think we should not have the app name in the tag for this

Reply:
blocker for this update?

Reply:
addressed for next release https://github.com/jenkinsci/custom-distribution-service/pull/164

Reply:
not a blocker